app/_data/cs.yml

```diff
@@ -78,8 +78,8 @@ nav:
     url:          "/cs/blog/"
   - text:         "PROZKOUMAT"
     links:
-      - text:     "STATISTIKY"
-        url:      "/cs/statistiky/"
+      - text:     "STATISTIKY <svg width='9' height='9' viewBox='0 0 11 11' title='externalLink' style='height: 9px;'><path fill='currentColor' d='M9.778 9.778H1.222V1.222H5.5V0H1.222C.544 0 0 .55 0 1.222v8.556C0 10.45.544 11 1.222 11h8.556C10.45 11 11 10.45 11 9.778V5.5H9.778v4.278zM6.722 0v1.222h2.194L2.91 7.23l.862.862 6.007-6.007v2.194H11V0H6.722z'></path></svg>"
```

Comment:
Did you test how this renders? Is there enough space in the navigation dropdown for this icon?

Reply:
![image](https://github.com/MissingMaps/missingmaps.github.io/assets/1847818/37276d6c-304f-4ed4-a712-19e8dd4d5627)
looks like this